src/styles/headerCell.css

```diff
@@ -0,0 +1,25 @@
+.cell {
+    composes: cell from './shared/cell.css';
+}
+
+.sortable {
+    cursor: pointer;
+}
+
+.sortArrow {
+    width: 16px;
+    height: 16px;
+    background-image: url('images/ui-icons_222222_256x240.png');
```

Comment:
Can we just set `content` to use a unicode arrow char (like ComboBox does) for now?